src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenSilentParameterBuilder.cs

```diff
@@ -80,6 +81,27 @@ public AcquireTokenSilentParameterBuilder WithForceRefresh(bool forceRefresh)
             return this;
         }
 
+        /// <summary>
+        /// To help with resiliency, AAD Cached Credential Service (CCS) operates as an AAD backup.
+        /// </summary>
+        /// <param name="userObjectIdentifier">GUID which is unique to the user, parsed from the client_info.</param>
+        /// <param name="tenantIdentifier">GUID format of the tenant ID, parsed from the client_info.</param>
+        /// <returns>The builder to chain the .With methods</returns>
+        public AcquireTokenSilentParameterBuilder WithCcsRoutingHint(string userObjectIdentifier, string tenantIdentifier)
```

Comment:
Is this needed on AcquireTokenSilent? MSAL already has the uid.tid from the account object, which is mandatory in this API.